index.html

```diff
@@ -785,6 +785,19 @@
           "site preferences".
         </p>
       </section>
+      <section>
+        <h3>
+          Managing source swapping between breakpoints
+        </h3>
+        <p>
+          When a new image is loaded via &lt;source&gt; after hitting a new
+          breakpoint, there is a delay while the image is loaded. Any CSS which
+          matches the breakpoint applies itself. However, we're stuck with the
+          previous image while the new one loads. This can result in blurry
+          imagery, or a larger image being scaled down in place of a smaller
+          one.
+        </p>
```

Comment:
Can you add a phrase with something like "Developers need means to control that flash of poor quality imagery in these cases"?